csatar-plugins/csatar/updates/builder_table_update_csatar_csatar_scouts_5.php

```diff
@@ -0,0 +1,27 @@
+<?php namespace Csatar\Csatar\Updates;
+
+use Schema;
+use October\Rain\Database\Updates\Migration;
+
+class BuilderTableUpdateCsatarCsatarScouts5 extends Migration
+{
+    public function up()
+    {
+        Schema::table('csatar_csatar_scouts', function($table)
+        {
+            $table->integer('special_diet')->unsigned();
+            $table->integer('religion')->unsigned();
+            $table->integer('tshirt_size')->unsigned();
```

Comment:
These columns are not needed. Please remove them with a new migration and don't edit old migration files.